clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -6155,6 +6155,44 @@ RValue CodeGenFunction::EmitBuiltinExpr(const GlobalDecl GD, unsigned BuiltinID,
     return RValue::get(Builder.CreateIntrinsic(
         llvm::Intrinsic::cheri_cap_perms_check, {SizeTy}, {Cap, Perms}));
   }
+  case Builtin::BI__builtin_cheriot_sealing_type: {
+    const auto *ArgLit = dyn_cast<StringLiteral>(E->getArg(0));
+    assert(ArgLit && "Argument to built-in should be a string literal!");
+    auto SealingTypeName = ArgLit->getString().str();
+    SealingTypeName.erase(
+        std::remove_if(SealingTypeName.begin(), SealingTypeName.end(), isspace),
+        SealingTypeName.end());
+
+    auto CompartmentName = getLangOpts().CheriCompartmentName;
+    auto PrefixedImportName =
+        CHERIoTSealingKeyTypeAttr::getSealingTypeSymbolName(CompartmentName,
+                                                            SealingTypeName);
+    auto *Mod = &CGM.getModule();
```

Comment:
No need to take the address here

Reply:
`= CGM.getModule()` uses a deleted constructor: 
```
llvm-project/clang/lib/CodeGen/CGBuiltin.cpp:6166:10: error: call to implicitly-deleted copy constructor of 'llvm::Module'
 6166 |     auto Mod = CGM.getModule();
 ```

Reply:
The way to do it is `auto &Mod = CGM.getModule();`